Support for custom `srcDir`

This currently gives:
```
ERROR in ./.nuxt-storybook/storybook/preview.js
Module not found: Error: Can't resolve '~/.nuxt-storybook/components'
```

--- a/docs/content/en/manual-setup.md
+++ b/docs/content/en/manual-setup.md
@@ -37,7 +37,7 @@ Note that if you want to use Nuxt features in your storybook (like store or axio
 
 Create another file called `preview.js` inside `storybook` directory and import Nuxt helpers in it.
 ```js{}[preview.js]
-import '~/.nuxt-storybook/storybook/preview.js'
+import '~~/.nuxt-storybook/storybook/preview.js'
 ```
 
 All done, just start storybook and make your changes.

--- a/examples/manual-setup/storybook/preview.js
+++ b/examples/manual-setup/storybook/preview.js
@@ -1,1 +1,1 @@
-import '~/.nuxt-storybook/storybook/preview.js'
+import '~~/.nuxt-storybook/storybook/preview.js'

--- a/storybook/preview.js
+++ b/storybook/preview.js
@@ -3,8 +3,8 @@
 <% } %>
 import Vue from 'vue'
 import '~storybook/mock'
-<% if (options.store) { %>import { createStore } from '~/.nuxt-storybook/store'<% } %>
-<% if (options.components) { %>import * as components from '~/.nuxt-storybook/components'<% } %>
+<% if (options.store) { %>import { createStore } from '~~/.nuxt-storybook/store'<% } %>
+<% if (options.components) { %>import * as components from '~~/.nuxt-storybook/components'<% } %>
 /* Plugins */
 <% options.plugins.forEach((plugin) => { %>import <%= plugin.name %> from '<%= plugin.name %>' // Source: <%= relativeToBuild(plugin.src) %> (mode: '<%= plugin.mode %>')
 <% }) %>

